install.sh

```diff
@@ -66,18 +66,20 @@ msg "Currently, you can install one of 'makedeb', 'makedeb-beta', or"
 msg "'makedeb-alpha'."
 msg "If you're not sure which one to choose, pick 'makedeb'."
 
-while true; do
-    read -p "$(question "Which release would you like? ")" response
+if [[ -z "$MAKEDEB_PKG" ]]; then
```

Comment:
My code style tends to be `${var:+x}` and `${var:-x}`, so I'd prefer if something like this was used:

```sh
if [[ "${MAKEDEB_PKG:-x}" == 'x' ]]; then
```

Reply:
Thanks for the review and the great software!

Pushed a new commit with ↓
```bash
if [[ "${MAKEDEB_PKG:-undefined}" == 'undefined' ]]; then
```

Please let me know if that's fine.